src/context/login-context.js

```diff
@@ -0,0 +1,28 @@
+import { useReducer, createContext, useContext } from "react";
+import { loginReducer } from "../reducers";
+
+const defaultLoginState = {
+	email: "",
+	password: "",
+	showPassword: false,
```

Comment:
I suggest using this locally within your login form. The reason is there's no need for showPassword to be a global state that needs to be common among all components.

Reply:
Yeah right it won't be accessed anywhere outside this page. Thanks, will make this change